packages/surveys/src/components/questions/OpenTextQuestion.tsx

```diff
@@ -130,8 +134,16 @@ export const OpenTextQuestion = ({
                 className="fb-border-border placeholder:fb-text-placeholder fb-bg-input-bg fb-text-subheading focus:fb-border-brand fb-rounded-custom fb-block fb-w-full fb-border fb-p-2 fb-shadow-sm focus:fb-ring-0 sm:fb-text-sm"
                 pattern={question.inputType === "phone" ? "[+][0-9 ]+" : ".*"}
                 title={question.inputType === "phone" ? "Please enter a valid phone number" : undefined}
+                minlength={question.inputType === "text" ? question.minLength || undefined : undefined}
+                maxlength={question.inputType === "text" ? question.maxLength || undefined : undefined}
               />
             )}
+            {question.inputType === "text" && question.maxLength !== undefined && (
+              <span
+                className={`fb-text-xs ${currentLength >= question.maxLength ? "fb-text-red-500 font-semibold" : "text-neutral-400"}`}>
```

Comment:
currentLength can never be greater than the maxLength; please fix this check to the `currentLength === question.maxLength`

Reply:
Yes, indeed! but I kept this for a corner case. If the max length set by the user is , let's say 200 initially and exhausts the limit, 200/200 will be shown in red color but again, if the user changes the max value to let's say 150, as the current length is still at 200, this 200/150 will be also shown in red color. I have attached loom video to demonstrate what I mean. Let me know your thoughts.